src/services/codefixes/helpers.ts

```diff
@@ -143,14 +143,15 @@ namespace ts.codefix {
                 break;
         }
 
-        function outputMethod(signature: Signature, modifiers: NodeArray<Modifier> | undefined, name: PropertyName, body?: Block): void {
-            const method = signatureToMethodDeclaration(context, signature, enclosingDeclaration, modifiers, name, optional, body, importAdder);
+        function outputMethod(signature: Signature, modifiers: NodeArray<Modifier> | undefined, name: PropertyName, preferences: UserPreferences, body?: Block): void {

```

Comment:
Nit: I think that `preferences` should be either the first or last parameter here

Reply:
Oke, I think it should be first because there is the last optional `body` parameter. Is it ok?

Reply:
Sure!